addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -1356,15 +1356,15 @@ const DisableOverlayButtonOption = options.Class.extend({
 // Disable duplicate button for model fields
 options.registry.WebsiteFormFieldModel = DisableOverlayButtonOption.extend({
     start: function () {
-        this.disableButton('clone', _t('You can\'t duplicate a model field.'));
+        this.disableButton('clone', _t('You cannot duplicate this field.'));
```

Comment:
This commit should be removed in stable IMHO.
It is better to keep a less clear wording but which is translated than a clearer wording that will be understand by only english people.
This commit will make it clearer for english speaking people but will make it not understandable to the rest of the world.
cc @pierre-pde let me know if you don't agree